src/containers/Login.vue

```diff
@@ -38,75 +38,59 @@
             <img
               style="position: relative"
               class="laptop"
-              src="@/assets/images/laptop.png"
+              src="@/assets/images/landing_new/laptop.svg"
               alt="laptop preview"
             />
           </div>
         </div>
       </div>
 
       <!--NEW WAY-->
-      <div class="new container p-0">
-        <h1 class="new-1 phonepad head">New Way to Track Your Requirements</h1>
+      <div class="new container p-0 new-way">
         <div class="row new no-gutters">
-          <div class="col-12 col-md-6 tasks-wrapper">
-            <div class="row tasks">
-              <div class="col-1 tasks">
-                <img src="@/assets/images/Task1.svg" alt="checklist" />
-              </div>
-              <div class="col-11">
-                <p class="sub sub--task">Fully personalized to track your requirements</p>
-              </div>
-            </div>
-            <div class="row tasks">
-              <div class="col-1 tasks">
-                <img src="@/assets/images/Task2.svg" alt="browser" />
-              </div>
-              <div class="col-11">
-                <p class="sub sub--task">Customizable interface to view your courses</p>
-              </div>
-            </div>
-            <div class="row tasks">
-              <div class="col-1 tasks">
-                <img src="@/assets/images/Task3.svg" alt="Network" />
-              </div>
-              <div class="col-11">
-                <p class="sub sub--task">Built-in system to check your progress</p>
-              </div>
-            </div>
-            <div class="row tasks">
-              <div class="col-1 tasks">
-                <img src="@/assets/images/Task4.svg" alt="Starred comment" />
-              </div>
-              <div class="col-11">
-                <p class="sub sub--task">Recommends courses based on your needs</p>
-              </div>
-            </div>
+          <div class="col tasks">
+            <figure class="figure">
```

Comment:
Huh, I'd never heard of `figure` or `figcaption` elements before. Just out of curiosity, is there a reason why you used these instead of `img` tags?

Reply:
The attach captions to images more easily.